android/app/src/main/java/org/fossasia/openevent/api/DataDownloadManager.java

```diff
@@ -16,7 +20,8 @@
 public final class DataDownloadManager {
     private static DataDownloadManager instance;
 
-    private APIClient client = new APIClient();
+    private APIClient openEventClient = new APIClient(true);
```

Comment:
the constructor will then be empty since the okHttpClientBuilder will be required for both. Correct me if I am wrong.